contracts/utils/Create2.sol

```diff
@@ -0,0 +1,53 @@
+pragma solidity ^0.5.0;
+
+/**
+ * @dev Helper to make usage of the `CREATE2` EVM opcode easier and safer.
+ * `CREATE2` can be used to compute in advance the address where a smart
+ * contract will be deployed, which allows for interesting new mechanisms known
+ * as 'counterfactual interactions'.
+ *
+ * See the https://eips.ethereum.org/EIPS/eip-1014#motivation[EIP] for more
+ * information.
+ */
+library Create2 {
```

Comment:
I'm not a fan of this name. Do we have other options?

Reply:
Usage will be `Create2.deploy()`. I also dislike the name, but I'm not sure how we can more succintly convey the underlying idea.